GSA-TTS/FAC
# Delete old replaced migrations

Finally finishes out #4259.

This PR deletes the old migrations - which should be fine since they have already been applied to production, and the squashed migration has been applied to production as well. The PR also removes the "replaces" call in the squashed migration to avoid referencing a deleted file.

## How to test

Run a clean version of the docker container, which should attempt to apply migrations. It should apply the squashed migration for the audit app and start the server with no issues.

## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
